src/RoleObjects/SliderData.js

```diff
@@ -1,115 +1,190 @@
+import KeyCodes from 'keycodes-enum';
 import _ from 'lodash';
 import InputTagData from './InputTagData';
 
 /**
  * Class for the slider role
  */
-
 export default class SliderData extends InputTagData {
   constructor(displayObject, role, domIdPrefix) {
     super(displayObject, role, domIdPrefix);
 
-    _.bindAll(this, 'onChange');
-    this._reactProps.onChange = this.onChange;
+    _.bindAll(this, '_onChange', '_onKeyDown');
+    this._reactProps.onChange = this._onChange;
+    this._reactProps.onKeyDown = this._onKeyDown;
     this._reactProps.type = 'range';
   }
 
-  onChange(evt) {
```

Comment:
This got moved down as a result of it being designated a private function and wanting to list the public members first.